src/main/java/org/openstreetmap/atlas/geography/sharding/README.md

```diff
@@ -0,0 +1,11 @@
+# Sharding
+
+Sharding is a way to split the world in small units of work.
```

Comment:
maybe something about they should be roughly equal sized pieces?

Reply:
Sounds good!